leaflet/admin.py

```diff
@@ -55,3 +56,6 @@ class LeafletMap(self.widget):
             map_height = self.map_height
             display_raw = self.display_raw
         return LeafletMap
+
+
+    change_list_template = 'leaflet/admin/change_list_geo.html'
```

Comment:
Since this is a rather radical change, I would do another admin class instead, at least the time to stabilize the whole thing